memberberries.json

```diff
@@ -26,7 +26,7 @@
     "when Homefront: Revolution was coming very soon to Linux",
     "when the Darksiders 1 & 2 ports were 98% finished",
     "when WB Games published Linux games",
-    "when we contemplated all the great games we would have with the Cry Engine"
+    "when we contemplated all the great games we would have with the Cry Engine",
     "when the SmachZ was announced and everyone got excited",
     "when 'Made to Wade' was on the Feral radar and meant they were porting Shadow Warrior 2"
```

Comment:
would a trailing comma here cause an error when loading the json list?
looks like a missed opportunity ?

Reply:
I am terrible. Understood.